focus/operations/src/project_cache.rs

```diff
@@ -32,13 +32,13 @@ pub fn push(
         .ok_or_else(|| anyhow::anyhow!("Project cache remote endpoint not configured"))?;
 
     let cache = ProjectCache::new(&repo, endpoint, app)?;
-    let keys = cache
+    let result = cache
         .generate_all(commit.id(), shard_index, shard_count)
         .context("Generating project cache data failed")?;
     let (_, build_graph_hash) = cache.get_build_graph_hash(commit.id(), true)?;
```

Comment:
Couldn't get to review the original project cache  in time but is there data that we're sending to TI so we have stats on the hit ratio?

Reply:
We have a field `sync_mechanism` indicating whether the sync operation was answered by the project cache. [See here](https://github.com/twitter/focus/search?q=sync_mechanism).